experimental/generation/generator/templates/library-Help.en-us.lg

```diff
@@ -1,32 +1,32 @@
 [library.en-us.lg]
 
-# askHelp
+# getAskHelpText
```

Comment:
Maybe getAutoHelpText?

Reply:
ok

---
In reply to: [433372587](https://github.com/microsoft/BotBuilder-Samples/pull/2461#discussion_r433372587) [](ancestors = 433372587)